services/search-builder.js

```diff
@@ -102,22 +102,22 @@ function SearchBuilder(model, opts, params, fieldNamesRequested) {
 
             var q = {};
             var columnName = field.columnName || field.field;
-            var column = opts.sequelize.col(association.as + '.' + columnName);
+            var column = opts.sequelizeFct.col(association.as + '.' + columnName);
 
             if (field.field === schemaAssociation.idField) {
               if (field.type === 'Number') {
                 var value = parseInt(params.search, 10) || 0;
                 if (value) {
-                  q = opts.sequelize.where(column, ' = ',
+                  q = opts.sequelizeFct.where(column, ' = ',
                     parseInt(params.search, 10) || 0);
                 }
               } else if (params.search.match(REGEX_UUID)) {
-                q = opts.sequelize.where(column, ' = ', params.search);
+                q = opts.sequelizeFct.where(column, ' = ', params.search);
               }
             } else if (field.type === 'String') {
-              q = opts.sequelize.where(
-                opts.sequelize.fn('lower', column), ' LIKE ',
-                opts.sequelize.fn('lower', '%' + params.search + '%')
```

Comment:
All theses identical changes, show that the current implementation is not that great.
In a perfect world, we should not have to replace all these `opts.sequelize` by `opts.sequelizeFct`.